src/config/excludedApps.yaml

```diff
@@ -0,0 +1 @@
+- settings
```

Comment:
Why a YAML and not a JSON file?

Reply:
We were already using those files for configs in cozy-bar so, it allows to add comments in configuration even if for here YAML or JSON doesn't matter